cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RemoteLauncher.java

```diff
@@ -51,7 +51,7 @@
    */
   public static void main(String[] args) throws Exception {
     if (args.length < 2) {
-      System.out.println("Usage: java " + TwillLauncher.class.getName() + " [mainClass] [use_classpath] [args...]");
+      System.out.println("Usage: java " + RemoteLauncher.class.getName() + " [mainClass] [use_classpath] [args...]");
```

Comment:
do we still need `RemoteLauncher`? Can we rely on `TwillLauncher` directly now that `TWILL-259` is fixed?
https://github.com/apache/twill/pull/89